packages/fast-components-styles-msft/src/hypertext/index.ts

```diff
@@ -1,30 +1,35 @@
-import designSystemDefaults, { DesignSystem } from "../design-system";
-import { ensureBrandNormal, ensureForegroundNormal } from "../utilities/colors";
+import { DesignSystem, withDesignSystemDefaults } from "../design-system";
+import {
+    accentFillRest,
+    neutralForegroundRest,
+    neutralOutlineHover,
+} from "../utilities/color";
 import {
     ComponentStyles,
     ComponentStyleSheet,
     CSSRules,
 } from "@microsoft/fast-jss-manager";
 import { HypertextClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
 import { applyFocusVisible } from "@microsoft/fast-jss-utilities";
-import typographyPattern from "../patterns/typography";
 
 const styles: ComponentStyles<HypertextClassNameContract, DesignSystem> = (
     config: DesignSystem
 ): ComponentStyleSheet<HypertextClassNameContract, DesignSystem> => {
+    const designSystem: DesignSystem = withDesignSystemDefaults(config);
+
     return {
         hypertext: {
             outline: "none",
             textDecoration: "none",
-            ...typographyPattern.rest,
+            color: neutralForegroundRest,
             "&:link, &:visited": {
-                borderBottom: `1px solid ${ensureBrandNormal(config)}`,
-                color: ensureBrandNormal,
+                borderBottom: `1px solid ${accentFillRest(designSystem)}`,
+                color: accentFillRest,
```

Comment:
You actually want to use the "accentForeground" recipe for these - and the states as appropriate (rest, hover, active)